examples/integration/BUILD

```diff
@@ -47,7 +47,7 @@ exports_files(["README.md"])
     xcodeproj(
         name = "xcodeproj-{}".format(simulator_cpu),
         associated_extra_files = ASSOCIATED_EXTRA_FILES,
-        build_mode = "xcode",
+        build_mode = "bazel",
```

Comment:
for testing

Reply:
You should be able to revert this and update the fixtures, right? Anything else missing before this is no longer a draft?

Reply:
Reverted. I updated the fixtures.

>Anything else missing before this is no longer a draft?
`DBGSourcePathRemapping`. 

And maybe some additional tests or is the current one enough?

Reply:
I can't think of what additional tests we could do at this point, but if you have ideas, I'm all for them.

Reply:
I was thinking about some kind of E2E test that would check that dSYM is present, but I don't see how to add this type of test right now. 

It might be better to implement the `DBGSourcePathRemapping` logic in a separate PR, so I'm going to mark this PR as "Ready for Review".

I've fixed some linter errors, so I think you can restart CI.